jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/Collections.java

```diff
@@ -1848,7 +1848,7 @@ public boolean equals(Object o) {
 
         SynchronizedCollection(Collection<E> c) {
             this.c = Objects.requireNonNull(c);
-            mutex = this;
+            mutex = new Object();
```

Comment:
I moved Collections. However, I don't see a similar folder structure for luni/LinkedBlockingQueue. Let me know if that also gets moved somewhere. Also moved LinkedBlockingQueueTest closer to where the LinkedListTest file is, but that may be incorrect. If there's a source org doc, let me know. Will take a look. Speaking of, if there's a "here's how we set up our build env" doc, that would be great too. My round trip is currently change file, 'make dist', and run in my downstream code.